### PR TITLE
Update the Java tools deploy jars.

### PR DESCRIPTION
**Pull request by iirina**
JacocoCoverageRunner needed to incorporate recent changes.

RELNOTES: None.

### CODE REVIEW

File: third_party/java/java_tools/README.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
-The following tools were built with bazel 0.22.0 at commit 569e2cb4d1b7653b5a02bbf664765476233e6c33
+
```
Comment (lfpino) [inline third_party/java/java_tools/README.md]: nit: Delete blank line

**Comment by iirina** [comment]
@lfpino Done.

**Comment by iirina** [comment]
@lfpino PTAL

**Comment by iirina** [comment]
Closed in favor of #7558 which also includes 1724cb27adc695e34ad59623201ecfabc1f45135

**Pull request closed without merging**